CMakeLists.txt

```diff
@@ -49,6 +49,10 @@ install(EXPORT valijsonConfig
     DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/valijson"
 )
 
+install(FILES bundle.sh
+    DESTINATION ${CMAKE_INSTALL_BINDIR}
+)
+  
```

Comment:
I'm not all that familiar with CMake. Do you know if we can rename this file as part of the installation process? My concern is that `bundle.sh` is very generic, and may conflict with other packages or scripts on the user's system. Whereas `valijson-bundle.sh` would be safer.

Reply:
@tristanpenman There is a `RENAME` option but I feel there is an element of obfuscation, why not name the original file as `valijson-bundle.sh` in the first place? Is there an operational/build requirement for it to be named `bundle.sh` ?

Reply:
That sounds like a good solution to me. Are you happy to push a rename change for this PR?